packages/eslint-config-typescript/index.js

```diff
@@ -51,6 +51,14 @@ module.exports = {
         leadingUnderscore: 'forbid',
         trailingUnderscore: 'forbid',
       },
+      {
+        selector: 'property',
```

Comment:
I think we can be stricter and use `objectLiteralProperty`.

`property` also matches `classProperty` and `typeProperty`, which I don't think need these?